src/main.js

```diff
@@ -108,7 +109,14 @@ app.use(VueRecaptchaPlugin, {
   v3SiteKey: '6Lc-LXsnAAAAAHGha6zgn0DIzgulf3TbGDhnZMAd',
 });
 
-initSentry(app);
+if (process.env.NODE_ENV === 'production') {
+  // Configure logging, and redirect console.log, console.warn, and console.error to the logger
+  const logger = log({ level: 'error' });
+  console.log = logger.info;
+
+  // Initialize Sentry if in a production environment
+  initSentry(app);
+}
```

Comment:
Even though using a logging library to suppress logging levels is a great idea, I would recommend against that approach as it can make debugging in remote environments so much more complicated. Theoretically the most important logs would be at the error log level, but not always. 

Since this would only be a concern in the CI environment (right?), can we not use `cy.on()` to intercept the console messages and then filter the messages for the debug token before they are being printed to the console? 

Alternatively, if you really want to control console logging, I'd recommend using the [debug](https://www.npmjs.com/package/debug) package. It's a great little tool that would let us use debug keys to control logging output. In other words: only those with the correct debug key in localStorage could see the console outputs. Wanted to write a ticket to recommend this package as it would also help bring some structure in the existing console logs.

Reply:
I would agree that it would be a good idea to use the cy.on() hook to filter out console logs that contain the debug token. But the issue is still that our PR links, which use the debug token, are logging the token in the console. These PR links are publicly accessible, since our dashboard repo is public.